integration/localnet/Makefile

```diff
@@ -1,5 +1,5 @@
 COLLECTION = 3
-CONSENSUS = 3
+CONSENSUS = 1
```

Comment:
is this for local testing, if so, better to change back after testing

Reply:
thanks, I forgot to switch it back